source/adapters/hip/context.cpp

```diff
@@ -47,18 +47,6 @@ UR_APIEXPORT ur_result_t UR_APICALL urContextCreate(
     // Create a scoped context.
     ContextPtr = std::unique_ptr<ur_context_handle_t_>(
         new ur_context_handle_t_{phDevices, DeviceCount});
-
```

Comment:
Why are you removing the base event?

Reply:
I moved the base event to the device to be consistent with CUDA adapter. This is a bit separate of adding the new profiling stream to both adapters but I combined it into single PR. The first 3 commits are only the work on HIP - moving `EvBase` to device and making it like it is in CUDA

Reply:
Aha yes I see. It looks like we were recording two base events for HIP?

Reply:
The base event for HIP used to belong to the `platform` and we were recording it twice, once here and once in `urPlatformGet`

Reply:
OK good catch